playback/tape_recorder.py

```diff
@@ -694,10 +706,14 @@ def decorated_function(*args, **kwargs):
                     try:
                         return self._playback_recorded_interception(interception_key, args, kwargs, data_handler)
                     except RecordingKeyError:
-                        if not run_intercepted_when_missing:
-                            raise
-                        # Run the original method when content was missing in recording
-                        return func(*args, **kwargs)
+                        if run_intercepted_when_missing:
```

Comment:
If run_intercepted_when_missing and value_when_missing are both true, should we throw an error to tell the user to pick only one? Or should we try one and then the other?

Reply:
I could have added an error but I don't think it is worth it.
Basically the run intercepted is taking precedence